packages/nx/src/adapter/ngcli-adapter.ts

```diff
@@ -15,13 +16,14 @@ import { FileBuffer } from '@angular-devkit/core/src/virtual-fs/host/interface';
 // Importing @angular-devkit/architect here will cause issues importing this file without @angular-devkit/architect installed
 /* eslint-disable no-restricted-imports */
 import type { Architect } from '@angular-devkit/architect';
+import { Target } from '@angular-devkit/architect';
```

Comment:
Non-type imports can bring back the issues that were addressed in https://github.com/nrwl/nx/pull/16595.
```suggestion
import type { Target } from '@angular-devkit/architect';
```

Reply:
no problem